lib/spec/sww.sw-spec.js

```diff
@@ -0,0 +1,71 @@
+importScripts('/base/lib/spec/nodeMock.js');
+importScripts('/base/lib/sww.js');
+importScripts('/base/lib/router.js');
+
+describe('ServiceWorkerWare', function () {
+  'use strict';
+
+  var worker;
+  var request;
+
+  var fallbackMW = function(req, res) {
```

Comment:
Will need to keep the if, as we still are using this model until your work lands ;P